lib/utils/proxyAgent.js

```diff
@@ -19,7 +19,12 @@ const { Agent } = require('agent-base');
  * @api public
  */
 class HttpsProxyAgent extends Agent {
-  constructor(_opts, initializationPromise) {
+  constructor(
+    _opts,
+    updateCertificateCallback,
```

Comment:
remove the name callback, its like calling a method `function`
```suggestion
    updateCertificate,
```